PlayCover.xcodeproj/project.pbxproj

```diff
@@ -983,12 +983,12 @@
 				minimumVersion = 1.0.0;
 			};
 		};
-		68E48BA029570A2B00C39879 /* XCRemoteSwiftPackageReference "swiftui-cached-async-image" */ = {
+		68E48BA029570A2B00C39879 /* XCRemoteSwiftPackageReference "CachedAsyncImage" */ = {
 			isa = XCRemoteSwiftPackageReference;
-			repositoryURL = "https://github.com/lorenzofiamingo/swiftui-cached-async-image";
+			repositoryURL = "https://github.com/bullinnyc/CachedAsyncImage";
```

Comment:
Since the image cache package is being changed, can you update the README to reflect this?